jvm/src/main/scala/io/glutenproject/extension/StrategyOverrides.scala

```diff
@@ -76,8 +69,26 @@ object JoinSelectionOverrides extends Strategy with JoinSelectionHelper with SQL
       if (GlutenConfig.getSessionConf.forceShuffledHashJoin) {
         // Force use of ShuffledHashJoin in preference to SortMergeJoin. With no respect to
         // conf setting "spark.sql.join.preferSortMergeJoin".
-        val leftBuildable = canBuildShuffledHashJoinLeft(joinType)
-        val rightBuildable = canBuildShuffledHashJoinRight(joinType)
+        val (leftBuildable, rightBuildable) = if (GlutenConfig.getConf.isClickHouseBackend) {
+          // Currently, ClickHouse backend can not support AQE, so it needs to use join hint
+          // to decide the build side, after supporting AQE, will remove this.
```

Comment:
What is the relationship between do not support AQE and needs to use join hint? I'm preparing to move JoinSelectionOverrides to ColumnarOverrides. Can you help explain why? @zzcclp 
Thank you.

Reply:
cc @zhztheplayer

Reply:
remove these comments please, it's not correct, now it already supports AQE.

Reply:
OK.